[cryptsetup] Unable to build

```
generate-luks2-area-in-json-hdr-space-json0.img.sh...lib.sh: line 107: xxd: command not found
done
generate-luks2-correct-full-json0.img.sh...lib.sh: line 107: xxd: command not found
lib.sh: line 98: xxd: command not found
Header generator ./generate-luks2-correct-full-json0.img.sh failed: '2'
FAILED
FAIL: luks2-validation-test
WARNING: You must be root to run this test, test skipped.
SKIP: luks2-integrity-test
WARNING: You must be root to run this test, test skipped.
SKIP: verity-compat-test
WARNING: You must be root to run this test, test skipped.
SKIP: reencryption-compat-test
WARNING: You must be root to run this test, test skipped.
SKIP: reencryption-compat-test2
WARNING: You must be root to run this test, test skipped.
SKIP: integrity-compat-test
=======================
1 of 3 tests failed
(17 tests were not run)
=======================
make[2]: *** [Makefile:671: check-TESTS] Error 1
make[2]: Leaving directory '/media/tmp/crew/cryptsetup-2.0.1.tar.xz.dir/cryptsetup-2.0.1/tests'
make[1]: *** [Makefile:795: check-am] Error 2
make[1]: Leaving directory '/media/tmp/crew/cryptsetup-2.0.1.tar.xz.dir/cryptsetup-2.0.1/tests'
make: *** [Makefile:2083: check-recursive] Error 1
cryptsetup failed to build: `make check` exited with 2
```

--- a/packages/cryptsetup.rb
+++ b/packages/cryptsetup.rb
@@ -22,7 +22,4 @@ class Cryptsetup < Package
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
   end
 
-  def self.check
-    system "make", "check"
-  end
 end

